doc/cli/npm-search.md

```diff
@@ -17,6 +17,20 @@ expression characters must be escaped or quoted in most shells.)
 
 ## CONFIGURATION
 
+### json
+
+* Default: false
+* Type: Boolean
+
+Output search results as a JSON array.
+
+### tab-separated
```

Comment:
Yeah, I think I would. It's fine if it stays tab-separated, but I don't think we need a separate format for tab-separated results just for this subcommand. Just out of curiosity, what happens if there are tabs embedded in the description?